src/packagedcode/rubygems.py

```diff
@@ -306,6 +311,7 @@ def build_rubygem_package(gem_data, download_url=None, package_url=None):
 
     declared_license = '\n'.join(declared_licenses) or None
 
+
```

Comment:
`declared_license = '\n'.join(declared_licenses) or None` 

.... this no longer makes sense. We want to explicitly keep a list of such licenses and no longer a string

Reply:
Fixed